config/samples/k6_v1alpha1_k6_with_volumeClaim.yaml

```diff
@@ -0,0 +1,16 @@
+apiVersion: k6.io/v1alpha1
+kind: K6
+metadata:
+  name: k6-sample
+  namespace: load-test
+spec:
+  parallelism: 4
+  script:
+    volumeClaim:
+      name: stress-test-volumeClaim
+      # test.js should exist inside /test/ folder.
+      # And, All the js files and directories test.js
+      # is importing from should be inside the same directory as well.
+      file: test.js
+  runner:
+    image: <custom-image>
```

Comment:
Custom image is not a requirement for `volumeClaim`: could you please remove it from the example? So that there's no confusion about it.

Reply:
@yorugac Thanks for noticing that, Removed it now!